src/main/java/org/opentripplanner/middleware/controllers/api/ApiController.java

```diff
@@ -217,8 +219,8 @@ private T getOne(Request req, Response res) {
     /**
      * HTTP endpoint to delete one entity specified by ID.
      */
-    private String deleteOne(Request req, Response res) {
-        long startTime = DateTimeUtils.currentTimeMillis();
+    private T deleteOne(Request req, Response res) {
+        long startTime = System.currentTimeMillis();
```

Comment:
Please use DateTimeUtils for getting the current time in case this is tested with mocked time.

Reply:
Oops, sorry. Missed that in the merge.